modules/nw-control-dns-records-public-hosted-zone-infoblox.adoc

```diff
@@ -53,20 +53,21 @@ spec:
         name: infoblox-credentials
       gridHost: ${INFOBLOX_GRID_PUBLIC_IP}
       wapiPort: 443
-      wapiVersion: "2.3.1"
+      wapiVersion: "2.3.1" <3>
   domains:
   - filterType: Include
     matchType: Exact
     name: test.example.com
   source:
-    type: OpenShiftRoute <3>
+    type: OpenShiftRoute <4>
     openshiftRouteOptions:
-      routerName: default <4>
+      routerName: default <5>
 ----
 <1> Specifies the External DNS name.
 <2> Defines the provider type.
-<3> You can define options for the source of DNS records.
-<4> If the source type is `OpenShiftRoute`, you can pass the OpenShift Ingress Controller name. External DNS selects the canonical hostname of that router as the target while creating CNAME record.
+<3> The currently supported WAPI versions are 2.3.1, 2.4, 2.5, 2.6, 2.6.1, 2.7, 2.7.1, 2.7.2, 2.7.3, 2.8, 2.9, and 2.9.1. Unless explicitly stated in the release notes, previously available WAPI versions are intended to remain accessible and operative with later versions.
```

Comment:
The Infoblox test instance we use is 8.6.x, so nomanally we are supposed to fine up to 2.12.2. See [the corresponding EDO PR](https://github.com/openshift/external-dns-operator/pull/229) and [NIOS 8.6.2 release notes doc](https://community.infoblox.com/cixhp49439/attachments/cixhp49439/IPAM/6153/2/NIOS_8.6.2_ReleaseNotesREVC.pdf) ("Changes to Infoblox API and Restful API (WAPI)" chapter).